### DISCUSSION TITLE
css 단위 설정 px vs rem

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#20] [03.Technical]
rem을 이용해서 `반응형`을 하기로 결정!

<img width="640" alt="스크린샷 2022-07-28 오후 5 51 32" src="https://user-images.githubusercontent.com/72402747/181464200-ad076e4e-329e-47a5-b5a9-e401ffe7f311.png">

```
하나 더 충고해 드리고 싶은 말은, em 단위를 쓰실 땐 상속에 따른 애매한 상황을 피하고 또 크기가 대칭적으로 변할 수 있도록, 지정한 요소의 폰트 크기는 꼭 rem 단위를 쓰시기 바랍니다.
```

[윤정님이 정해주신 반응형 폰트 적용하기](https://firsteast.tistory.com/123)

[다은님이 주신 지은 멘토님이 주신 tv는 왜 소형인가요?](https://docs.microsoft.com/ko-kr/windows/apps/design/layout/screen-sizes-and-breakpoints-for-responsive-design)

**Comment by Hyevvy** [comment]
rem을 통해서 반응형을 하기로 결정 땅땅땅 🫠

**Comment by gitul0515** [comment]
padding, margin 등에는 rem을 사용하면 안 됩니다. 

크롬에서 폰트사이즈와 화면을 확대/축소 할 수 있는 설정이 있는데,
이렇게 되면 단순히 폰트 크기만 변경 했을 뿐인데
margin / padding 값이 커져서 데스크탑에서 모바일화면으로 보게 되는 현상이 발생 할 수 있습니다.

rem은 font-size에만 사용해야 할 것 같습니다. 

<a href="https://simsimjae.medium.com/%ED%81%AC%EB%A6%AC%EC%8A%A4%EB%A7%88%EC%8A%A4%EC%97%90-%EC%BD%94%EB%94%A9%ED%95%98%EA%B3%A0-%EC%9E%88%EB%8A%94-%EB%82%B4-%EC%9D%B8%EC%83%9D%EC%9D%B4-%EB%A0%88%EC%A0%84%EB%93%9C-%EA%B0%99%EC%A7%80%EB%A7%8C-%EC%98%A4%EB%8A%98%EB%8F%84-%EB%8B%AC%EB%A6%B0%EB%8B%A4-96f79ee41464">참고자료</a>
